include/Note.h

```diff
@@ -246,7 +259,10 @@ class LMMS_EXPORT Note : public SerializingObject
 	panning_t m_panning;
 	TimePos m_length;
 	TimePos m_pos;
-	DetuningHelper * m_detuning;
+	DetuningHelper * m_detuning = nullptr;
```

Comment:
Since I remember you saying you did not intend to directly change ``m_detuning`` (which I mistakenly understood you did), you can omit the in-class initialization and undo the changes you made to this line. This is so that the PR is more focused.